src/components/ChatDropdown/ChatDropdown.tsx

```diff
@@ -1,114 +1,70 @@
 import React from "react";
 import { Actions, TaskContext } from "@twilio/flex-ui";
-import Grid from "@material-ui/core/Grid";
-import Button from "@material-ui/core/Button";
-import Menu from "@material-ui/core/Menu";
-import MenuItem from "@material-ui/core/MenuItem";
-import ChatIcon from "@material-ui/icons/ChatBubble";
-import CircularProgress from "@material-ui/core/CircularProgress";
-import Typography from "@material-ui/core/Typography";
-import { withStyles } from "@material-ui/core/styles";
-
+import { Box } from '@twilio-paste/core/box';
+import { Text } from '@twilio-paste/text'
+import { Button } from '@twilio-paste/core/button'
+import {
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuGroup,
+  useMenuState
+} from '@twilio-paste/core/menu';
+import { ChatIcon } from "@twilio-paste/icons/esm/ChatIcon";
 import { useResponses } from "../../api";
 
-interface OwnProps {
-  // Props passed directly to the component
-  classes: any;
-}
-
-// Props should be a combination of StateToProps, DispatchToProps, and OwnProps
-type Props = OwnProps;
-
-const styles = {
-  root: {
-    padding: "24px",
-  },
-  icon: {
-    marginRight: 6,
-    fontSize: 14,
-  },
-  button: {
-    marginRight: 8,
-    backgroundColor: "rgb(5, 125, 158)",
-    color: "white",
-    width: "100%",
-  },
-};
-
-const ChatDropdown: React.FunctionComponent<Props> = (props: Props) => {
-  const { classes } = props;
-  const [anchorEl, setAnchorEl] = React.useState(null);
+const ChatDropdown: React.FunctionComponent = () => {
   const { status, data } = useResponses();
 
-  const handleClick = (event: any) => {
-    setAnchorEl(event.currentTarget);
-  };
-
-  const handleClose = () => {
-    setAnchorEl(null);
-  };
-
   const insertMessage = (conversationSid: string | undefined, text: any) => {
     if (!conversationSid) return;
     Actions.invokeAction("SetInputText", { body: text, conversationSid });
-    handleClose();
   };
 
+  const menu = useMenuState();
+
   return (
-    <Grid container className={classes.root}>
+    <Box paddingBottom="space60" margin="auto">
       <TaskContext.Consumer>
         {(context) => (
           <>
             {status === "loading" ? (
-              <CircularProgress className={classes.progress} />
+              <Button variant="primary" loading>Loading</Button>
```

Comment:
Should we switch this `Button` component to a `SkeletonLoader` for the loading state?

Reply:
Done!